understanding/20/focus-order.html

```diff
@@ -63,6 +83,8 @@ <h2>Intent of Focus Order</h2>
             operability only when navigation sequences affect meaning and operability.
          </li>
          
+         <li>Focus order does not need to follow visual presentation or layout.</li>
```

Comment:
I don't think that I agree with this statement. The issue is that the interpretation of the visual presentation is subjective.

Reply:
does not *necessarily* need to follow visual presentation, perhaps? i've seen extreme cases where auditors gleefully fail things if they're not exactly in the same order as the visual, with things like three little icon-based buttons that receive focus right-to-left instead of left-to-right (because the author used CSS floats that effectively reversed the order) ... but where the meaning was not impacted and it was not really a problem.

there must be some way we can somehow make that clear...

Reply:
wonder if it's too handwavy though. maybe best to expand this, or add some more words that reflect what @awkawk rightly says...that it's subjective?

Reply:
"Who is deciding what is the visual presentation order?" not sure i follow...is this rhetorical? the author/designer will work on the visual/layout, but for various reasons the focus order may be slightly different. the point is that as long as meaning/operation aren't affected (and it's...subjectively...not too haphazard jumping all over the place) it's not necessarily a failure if the focus order doesn't strictly follow the western top-to-bottom/left-to-right (in left-to-right writing cultures)